pkg/cmd/pulumi/util.go

```diff
@@ -1115,3 +1115,22 @@ func missingNonInteractiveArg(args ...string) error {
 			strings.Join(args[:len(args)-1], ", "), args[len(args)-1])
 	}
 }
+
+func promptUser(msg string, options []string, defaultOption string, opts display.Options) string {
```

Comment:
The only thing this uses from display.Options is `Color`, feels like this should just take the Color option not the whole display opts.

Reply:
That also makes the edits to state_edit smaller.